### PR TITLE
[FR] Replay

### PR DESCRIPTION
**Pull request by Ganondorfosu**
French translation of osu! wiki - Replay page

### CODE REVIEW

File: wiki/Replay/fr.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+![](img/Replay_failed.jpg "Sur la gauche se trouve un écran de Game Over normal, sur la gauche se trouve l'écran de Game Over sur un replay")
+
+Pour accéder à cet écran, échouez la map en vidant entière la barre de vie, ou ne dépassez pas + de 50% de la barre de vie avant la fin de map en osu!taiko.
+
+Sur l'écran de Game Over, appuyez sur la touche `F1` pour voir un replay de la partie perdue et appuyez sur la touche `F2` pour en sauvegarder un replay.
+Le replay se termine lorsque l'écran s'assombrit, avec le osu!logo au milieu et remplace l'écran de Game Over.
+
+Puisqu'il ne s'agit pas d'un écran de Game Over normal, le seul moyen de retourner sur l'écran de sélection de beatmap est d'appuyer sur la touche `Echap`.
+Trying to restart the beatmap to directly play the beatmap again in this state is not possible.
```
Comment (DoryanR) [inline wiki/Replay/fr.md:145]:
```suggestion
Essayer de redémarrer la beatmap pour le relancer directement dans cet état n'est pas possible.
```

File: wiki/Replay/fr.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+Après cela, appuyez sur `F9` pour accéder à la console de chat étendue et choisissez le jouer à observer.
+Vous devez avoir la **même** difficulté/beatmap que le joueur, ou un message d'avertissement apparaîtra en bas à droite pour vous dire que vous ne possédez pas la beatmap, vous ne pouvez donc pas obtenir le replay.
+Si vous possédez la beatmap, le replay en mode spectateur commencera.
+
+Un onglet `#spectator` s'ouvrira dans la console de chat (pour les spectateurs et le joueur) pour discuter de la partie du joueur.
+
+Notez que selon les paramètres du joueur, la liste des pseudos des spectateurs apparaîtra en blanc sur l'écran du joueur dans le coin supérieur gauche.
+
+Format et exemple de la liste que le joueur voit:-
```
Comment (DoryanR) [inline wiki/Replay/fr.md:177]:
```suggestion
Format et exemple de la liste que le joueur voit :
```

File: wiki/Replay/fr.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+
+Exemple:-
+Spectateurs (2):
+deadbeat
+ztrot
+```
+
+Un texte blanc défilant de droite à gauche apparaîtra lorsque vous regardez un replay en mode spectateur.
+
+Format exemple de ce texte:-
```
Comment (DoryanR) [inline wiki/Replay/fr.md:192]:
```suggestion
Format et exemple de ce texte :
```

File: wiki/Replay/fr.md [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+
+### General
+
+La vitesse du replay n'affecte pas la vitesse de défilement de la notice `REPLAY MODE/SPECTATOR MODE` .
+
+Les pauses ne seront pas comptées dans les données du replay; seul le gameplay actif est compté.
+
+### Spectateur
+
+Bien qu'il soit possible d'exporter un replay en mode spectateur, c'est une mauvaise si vous avez rejoint en plein milieu ou à la fin de la map.
```
Comment (DoryanR) [inline wiki/Replay/fr.md:244]:
```suggestion
Bien qu'il soit possible d'exporter un replay en mode spectateur, c'est une mauvaise idée si vous avez rejoint en plein milieu ou à la fin de la map.
```

File: wiki/Replay/fr.md [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+
+### Partie fictive
+
+![](img/Replay_preview_result.jpg "Exemple d'écran de résultat d'une partie fictive")
+
+Selon le moment aléatoire choisi par Auto, il pourrait commencer à la fin de la map sans le moindre objet, menant directement à l'écran de résultat avec 0 score, 0 combo, un rang D et 100% de précision
+Lorsque vous regardez le replay, la perte de vie fonctionnera comme d'habitude, pas de mouvement de curseur, et pas de miss (d'où les 100% de précision) jusqu'au moment où Auto a commencé à jouer après avoir appuyé sur `Voir le skin dans une partie fictive`, les objets commenceront à fonctionner
+
+Puisqu'Auto ne peut pas échouer une map, une barre de vie vide n'a pas d'effet sur Auto.
+Cependant, en osu!taiko, si Auto ne peut pas remplir la barre de vie à au moins 50%, le replay sera bloqué dans une _boucle infinie_.
```
Comment (DoryanR) [inline wiki/Replay/fr.md:268]: Il faudra rajouter une ligne vide en fin de document, car cela bloque la vérification de GitHub.

**Review by DoryanR — CHANGES_REQUESTED** [review]
After changes, it will be good for me.

File: wiki/Replay/fr.md [hunk 6]
```diff
@@ -0,0 +1,268 @@
+---
+needs_cleanup: true
+outdated: true
+---
```
Comment (DoryanR) [inline wiki/Replay/fr.md:4]: Fait un saut de page entre "---" et le TODO, car cela bloque la vérification de GitHub.

**Comment by DoryanR** [comment]
Also, for each italicization (with the `_`), you have to use `*` by putting this at the beginning of the place where you have to italicize, and at the end too, as `_` doesn't work, for conformity and formatting concerns between articles I guess.

File: wiki/Replay/fr.md [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+### Local (Solo)
+
+![](img/Replay_Solo.jpg "Un replay local")
+
+_Un replay local typique_
+
+Une partie locale en _Solo_.
+Tant que la difficulté a été réussie, un replay sera généré et pourra être exporté dans le dossier `Replays`.
+
+Dans le backend, une paire de fichiers sera créé dans le dossier caché, `Data/r`: `.osr` (osu! replay; le fichier du replay), et `.osg` (précision et combo en temps réel: utilisé uniquement pour les replays en mode spectateur) avec le même nom de fichier crypté.
```
Comment (DoryanR) [inline wiki/Replay/fr.md:124]:
```suggestion
Dans le backend, une paire de fichiers sera créé dans le dossier caché, `Data/r`: `.osr` (osu! replay ; le fichier du replay), et `.osg` (précision et combo en temps réel : utilisé uniquement pour les replays en mode spectateur) avec le même nom de fichier crypté.
```

File: wiki/Replay/fr.md [hunk 8]
```diff
@@ -0,0 +1,269 @@
...
+Il est hautement recommandé  de ne **pas renommer le fichier crypté**, et d'utiliser la touche `F2` à la place.
+
+Supprimer le fichier `.osg` n'a pas d'effet (il s'agit du reste du replay en spectateur, peut être supprimé en toute sécurité).
+Supprimer le fichier `.osr`  vous fera perdre le replay _pour toujours_ (puisque les données du replay ne sont plus présentes).
+Par ailleurs, supprimer le fichier `scores.db` (qui contient tous les scores locaux et les chemins vers les replays cryptés) _vous fera perdre tous les scores locaux et replays (non exportés) pour toujours_.
+
+Pour établir un pseudo sur des scores locaux en étant hors ligne (compte _Guest_), défilez vers le bas sur l'écran de résultat pour accéder aux résultats en ligne après avoir terminé une map et tapez le pseudo choisi dans la boîte de texte _Guest player name_.
+Sur cet écran, une alternative à la touche d'exportation `F2`  est d'activer le bouton `Save replay to Replays folder` situé sur la partie supérieure droite.
```
Comment (DoryanR) [inline wiki/Replay/fr.md:132]:
```suggestion
Sur cet écran, une alternative à la touche d'exportation `F2` est d'activer le bouton `Save replay to Replays folder` situé sur la partie supérieure droite.
```

File: wiki/Replay/fr.md [hunk 9]
```diff
@@ -0,0 +1,269 @@
...
+
+Retournez sur l'écran de sélection de beatmap et les changements seront mis à jour pour ce replay (Si aucun nom n'a été donné, il sera vide).
+
+### Echec
```
Comment (DoryanR) [inline wiki/Replay/fr.md:136]:
```suggestion
### Échec
```

File: wiki/Replay/fr.md [hunk 10]
```diff
@@ -0,0 +1,269 @@
...
+
+Sur l'écran de Game Over, appuyez sur la touche `F1` pour voir un replay de la partie perdue et appuyez sur la touche `F2` pour en sauvegarder un replay.
+Le replay se termine lorsque l'écran s'assombrit, avec le osu!logo au milieu et remplace l'écran de Game Over.
+
+Puisqu'il ne s'agit pas d'un écran de Game Over normal, le seul moyen de retourner sur l'écran de sélection de beatmap est d'appuyer sur la touche `Echap`.
+Essayer de redémarrer la beatmap pour le relancer directement dans cet état n'est pas possible.
+
+Cela ne fonctionne actuellement qu'en `Solo`.
+
+Voir un replay échoué ne fonctionnera pas avec les mods [No Fail(NF)](/wiki/Game_modifier/No_Fail)/[Relax(RL)](/wiki/Game_modifier/Relax)/[Auto Pilot(AP)](/wiki/Game_modifier/Autopilot) puisqu'il est impossible d'échouer.
```
Comment (DoryanR) [inline wiki/Replay/fr.md:150]:
```suggestion
Voir un replay échoué ne fonctionnera pas avec les mods [No Fail(NF)](/wiki/Game_modifier/No_Fail)/[Relax(RL)](/wiki/Game_modifier/Relax) et [Auto Pilot(AP)](/wiki/Game_modifier/Autopilot) puisqu'il est impossible d'échouer.
```

File: wiki/Replay/fr.md [hunk 11]
```diff
@@ -0,0 +1,269 @@
...
+Si vous possédez la beatmap, le replay en mode spectateur commencera.
+
+Un onglet `#spectator` s'ouvrira dans la console de chat (pour les spectateurs et le joueur) pour discuter de la partie du joueur.
+
+Notez que selon les paramètres du joueur, la liste des pseudos des spectateurs apparaîtra en blanc sur l'écran du joueur dans le coin supérieur gauche.
+
+Format et exemple de la liste que le joueur voit :
+
+```
+Format :-
```
Comment (DoryanR) [inline wiki/Replay/fr.md:181]:
```suggestion
Format :
```

File: wiki/Replay/fr.md [hunk 12]
```diff
@@ -0,0 +1,269 @@
...
+Spectateurs (#):
+{Pseudos des joueurs}
+
+Exemple:-
```
Comment (DoryanR) [inline wiki/Replay/fr.md:185]:
```suggestion
Exemple :
```

File: wiki/Replay/fr.md [hunk 13]
```diff
@@ -0,0 +1,269 @@
...
+
+Un onglet `#spectator` s'ouvrira dans la console de chat (pour les spectateurs et le joueur) pour discuter de la partie du joueur.
+
+Notez que selon les paramètres du joueur, la liste des pseudos des spectateurs apparaîtra en blanc sur l'écran du joueur dans le coin supérieur gauche.
+
+Format et exemple de la liste que le joueur voit :
+
+```
+Format :-
+Spectateurs (#):
```
Comment (DoryanR) [inline wiki/Replay/fr.md:182]:
```suggestion
Spectateurs (#) :
```

File: wiki/Replay/fr.md [hunk 14]
```diff
@@ -0,0 +1,269 @@
...
+{Pseudos des joueurs}
+
+Exemple:-
+Spectateurs (2):
```
Comment (DoryanR) [inline wiki/Replay/fr.md:186]:
```suggestion
Spectateurs (2) :
```

File: wiki/Replay/fr.md [hunk 15]
```diff
@@ -0,0 +1,269 @@
...
+Pour regarder un replay du serveur, vous devez avoir un compte connecté à Bancho.
+Sur l'écran de sélection de beatmap en `Solo`, allez sur le `Classement général` et cliquez sur le joueur que vous souhaitez pour regarder le replay.
+Cliquez sur le bouton `Watch replay` et osu! téléchargera le replay sur le serveur comme montré sur l'image de gauche.
+Le replay ne se joue qu'une fois et sera supprimé une fois quitté/fini.
+
+## Trivia
+
+### General
+
+La vitesse du replay n'affecte pas la vitesse de défilement de la notice `REPLAY MODE/SPECTATOR MODE` .
```
Comment (DoryanR) [inline wiki/Replay/fr.md:239]:
```suggestion
La vitesse du replay n'affecte pas la vitesse de défilement de la notice `REPLAY MODE/SPECTATOR MODE`.
```

File: wiki/Replay/fr.md [hunk 16]
```diff
@@ -0,0 +1,269 @@
...
+
+Lorsque vous regardez un replay depuis le classement (un replay exporté ne compte pas), le nombre affiché à côté dans le compteur de "Replays regardés par les autres" sur le profil du joueur du replay augmentera de 1.
+Si un nouveau score s'ajoute au top 1000 du classement, le replay de l'ancien détenteur du 1000ème score sera supprimé.
+
+Pour regarder un replay du serveur, vous devez avoir un compte connecté à Bancho.
+Sur l'écran de sélection de beatmap en `Solo`, allez sur le `Classement général` et cliquez sur le joueur que vous souhaitez pour regarder le replay.
+Cliquez sur le bouton `Watch replay` et osu! téléchargera le replay sur le serveur comme montré sur l'image de gauche.
+Le replay ne se joue qu'une fois et sera supprimé une fois quitté/fini.
+
+## Trivia
```
Comment (DoryanR) [inline wiki/Replay/fr.md:235]:
```suggestion
## Anecdotes
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
Still some things to work on.

File: wiki/Replay/fr.md [hunk 17]
```diff
@@ -0,0 +1,269 @@
...
+Par défaut, sans tag osu!supporter actif, la couleur des commentaires dépendra de la couleur affichée dans le chat (les comptes sans osu!supporter sont affichés en blanc) mis à part le créateur de la map (commentaires en bleu-clair).
+
+### Collections
+
+Ajouter cette difficulté/beatmap aux _Collections_.
+Ce bouton ouvrira le menu des collections pour ajouter la difficulté/beatmap en cours dans une des collections ou renommer les collections.
+
+Le replay ne se mettra pas en pause et continuera en arrière-plan sur ce menu.
+
+### Playback Speed
```
[the remark this answers is not in the document]
Comment (Ganondorfosu) [reply wiki/Replay/fr.md:75]: Je le laisse comme ça parce que c'est ce qui est écrit en jeu (même avec le jeu en français)

[the remark this answers is not in the document]
Comment (DoryanR) [reply wiki/Replay/fr.md:75]: Ok pas de soucis

**Review by DoryanR — APPROVED** [review]
After checking the file, I approve the changes.

**Review by TicClick — APPROVED** [review]
in future, please avoid translating articles that have `needs_cleanup` and `outdated` tags. it's very often a wasted effort, because you'll have to translate them twice.

**Pull request merged**